core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -200,7 +210,101 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
   }
 
   @Override
-  public void close() throws IOException {
+  public void createNamespace(Namespace namespace, ImmutableMap<String, String> meta) {
```

Comment:
This should use `Map` and not `ImmutableMap`. We don't want to leak Guava classes through this API.

Reply:
OK